engines/wintermute/base/base_file_manager.cpp

```diff
@@ -372,6 +373,10 @@ uint32 BaseFileManager::getPackageVersion(const Common::String &filename) {
 
 //////////////////////////////////////////////////////////////////////////
 bool BaseFileManager::hasFile(const Common::String &filename) {
+	Common::String backwardSlashesPath = filename;
```

Comment:
missing ENABLE_WME3D

Reply:
Done

Reply:
I'd actually suggest not having that macro-guard here, the fix should be applicable to all games (and makes the behaviour consistent with the file loading).